src/test/java/com/example/DoughnutShopTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DoughnutShopTest {
+
+    @Test
+    public void testPrepareDoughnut_forChocolate() {
+        DoughnutShop shop = new DoughnutShop();
+
+        String result = shop.prepareDoughnut("Chocolate");
+
+        assertEquals("A delicious chocolate doughnut prepared!", result);
+    }
+
+    @Test
+    public void testPrepareDoughnut_forCinnamon() {
```

Comment:
If `doughnutType` was a `HashMap<Enum, String>` then we could save these tests and check all cases with a single foreach